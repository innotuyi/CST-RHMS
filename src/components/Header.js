import React from "react";
import { Navbar, Nav } from "react-bootstrap";
import { Link } from "react-router-dom";

function Header() {
  return (
    <Navbar
      sticky="top"
      collapseOnSelect
      expand="lg"
      bg="white"
      className="shadow-sm mb-5 bg-white rounded"
    >
      <Navbar.Brand href="/">
        <img
          alt=""
          src="https://registration.ur.ac.rw/partnership/sites/default/files/UR%20LOGO%20FOR%20REGISTRATION%20SITE.JPG"
      
          height="80px"
          object-fit="contain"
        />
      </Navbar.Brand>
      <Navbar.Toggle aria-controls="responsive-navbar-nav" />
      <Navbar.Collapse id="responsive-navbar-nav">
        <Nav className="ml-auto mr-5">
          <Link className="nav-link" to="/login">
            Login
          </Link>
          <Nav.Link e href="">
            SignUp
          </Nav.Link>
        </Nav>
      </Navbar.Collapse>
    </Navbar>
  );
}

export default Header;
