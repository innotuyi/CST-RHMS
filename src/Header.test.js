import React from "react";
import Header from "./components/Header";

import { configure,shallow, mount } from "enzyme";
import toJSON from "enzyme-to-json";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

let wrapper;
beforeEach(() => {
   wrapper = shallow(<Header />);
})

describe("Header Tests", () => {
  it("matches Snapshot Collectly", () => {
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
