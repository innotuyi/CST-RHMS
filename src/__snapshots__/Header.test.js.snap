// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Tests matches Snapshot Collectly 1`] = `
<Navbar
  bg="white"
  className="shadow-sm mb-5 bg-white rounded"
  collapseOnSelect={true}
  expand="lg"
  sticky="top"
  variant="light"
>
  <NavbarBrand
    href="/"
  >
    <img
      alt=""
      height="80px"
      object-fit="contain"
      src="https://registration.ur.ac.rw/partnership/sites/default/files/UR%20LOGO%20FOR%20REGISTRATION%20SITE.JPG"
    />
  </NavbarBrand>
  <NavbarToggle
    aria-controls="responsive-navbar-nav"
    label="Toggle navigation"
  />
  <NavbarCollapse
    id="responsive-navbar-nav"
  >
    <Nav
      className="ml-auto mr-5"
      fill={false}
      justify={false}
    >
      <Link
        className="nav-link"
        to="/login"
      >
        Login
      </Link>
      <NavLink
        as={
          Object {
            "$$typeof": Symbol(react.forward_ref),
            "render": [Function],
          }
        }
        disabled={false}
        e={true}
        href=""
      >
        SignUp
      </NavLink>
    </Nav>
  </NavbarCollapse>
</Navbar>
`;
